addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -34,15 +33,16 @@
                     <div t-if="attachment.uploading" class="position-absolute top-0 bottom-0 start-0 end-0 d-flex align-items-center justify-content-center" title="Uploading">
                         <i class="fa fa-spin fa-spinner"/>
                     </div>
-                    <div class="position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end flax-wrap flex-column">
-                        <div t-if="showDelete and !ui.isSmall"
-                             class="btn btn-sm btn-dark rounded opacity-75 opacity-100-hover"
+                    <div class="position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end flax-wrap flex-column gap-1">
+                        <button t-if="showDelete and !ui.isSmall"
+                             class="btn btn-sm rounded opacity-75 opacity-100-hover px-1"
+                             t-att-class="store.theme === 'dark' ? 'btn-light' : 'btn-dark'"
```

Comment:
I'm not sure about this. `.btn-light` should support the dark theme, same for `.btn-dark`, we don't want to tweak the UI based on javascript conditions. 
If they don't, maybe it's not the best class for this button, `btn-secondary` might be an option.

Reply:
`btn-secondary` looks somewhat like a constant `btn-light` of the light theme.
What I'm trying to achieve here is keeping a `btn-dark` of the light theme.

AFAIK there's no such existing styling that is preserved between dark and white theme. `-dark` and `-light` modifiers are theme-dependent, which sucks.